modules/applications/pages/faq.adoc

```diff
@@ -142,21 +142,44 @@ Yes they are.
 === Is Bonita UI Builder Bonita Cloud compatible? 
 Yes it is. Reach out to your customer service representative to explore how you can migrate to Bonita Cloud.
 
+[.troubleshooting-title]
+== image:ROOT:images/troubleshooting.png[troubleshooting-icon] Troubleshooting
```

Comment:
```suggestion
== Troubleshooting
```

Reply:
Why not the icon like in most other places ?

Reply:
OK. Because most other places don't use the [.troubleshooting-title] which already adds the icon.
✅